packages/lang/llvm/package.mk

```diff
@@ -82,14 +47,32 @@ PKG_CMAKE_OPTS_TARGET="-DCMAKE_BUILD_TYPE=MinSizeRel \
                        -DLLVM_ENABLE_TERMINFO=OFF \
                        -DLLVM_ENABLE_ASSERTIONS=OFF \
                        -DLLVM_ENABLE_WERROR=OFF \
-                       -DLLVM_TARGET_ARCH="$TARGET_ARCH" \
                        -DLLVM_ENABLE_ZLIB=ON \
                        -DLLVM_BUILD_LLVM_DYLIB=ON \
                        -DLLVM_LINK_LLVM_DYLIB=ON \
                        -DLLVM_OPTIMIZED_TABLEGEN=ON \
-                       -DLLVM_TABLEGEN=$TOOLCHAIN/bin/llvm-tblgen \
                        -DLLVM_APPEND_VC_REV=OFF"
 
+PKG_CMAKE_OPTS_HOST="$PKG_CMAKE_OPTS_COMMON \
+                     -DLLVM_ENABLE_ZLIB=OFF \
+                     -DCMAKE_INSTALL_RPATH=$TOOLCHAIN/lib"
+
+make_host() {
+  ninja llvm-config llvm-tblgen
+}
+
+makeinstall_host() {
+  cp -a bin/llvm-config $TOOLCHAIN/llvm-config-host
```

Comment:
Worth keeping the s/SYSROOT_PREFIX/usr/bin//TOOLCHAIN/ change as separate commit, alongside the Mesa update?